### DISCUSSION TITLE
Nextcloud auto logoff feature logs off too soon

### DISCUSSION THREAD

**Discussion by roelofz** [nextcloud/all-in-one#5170] [Questions] [answered]
### Steps to reproduce
1. Configure config.php as follows:
  'remember_login_cookie_lifetime' => '43200',
  'session_lifetime' => '43200',
  'session_keepalive' => 'true',
  'auto_logout' => true,
Tested with multiple browsers (Edge and Firefox, no tracking protection for the nextcloud interface.
With the default settings, the session does not close

### Expected behavior
The session should be closed after 12 hours

### Actual behavior
The session is closed after less then half an hour
In some case it was possible after more then 12 hours to go back to the last page (Snappymail) without reauthenticating

### Host OS 
Ubuntu 22.04.4
#### Docker run command or docker-compose file that you used
    nextcloud-aio-mastercontainer:
      image: nextcloud/all-in-one:latest
      init: true
      container_name: nextcloud-aio-mastercontainer
      restart: unless-stopped
      volumes:
        - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
        - /var/run/docker.sock:/var/run/docker.sock:ro
      ports:
        - 8091:8080

      environment:
        - APACHE_PORT=11000
        - APACHE_IP_BINDING=0.0.0.0
        - NEXTCLOUD_DATADIR=/mnt/nextcloud
        - NEXTCLOUD_MOUNT=/mnt/nc-filedata
        - NEXTCLOUD_MEMORY_LIMIT=1024M
        - AIO_COMMUNITY_CONTAINERS=local-ai
      networks:
        nextcloud:
        frontend:

#### Nextcloud AIO version 
[Nextcloud Hub 8](https://nextcloud.com/) (29.0.4)
I think this needs some rework and may be related to AIO as multiple posts can be found of users needing help (and it seems fixed for some users, no working solutions appear in topics on help.nextcloud.com working in this setup.

**Comment by szaimen** [comment]
Hi, please open a thread at https://help.nextcloud.com for this

**Reply by roelofz** [reply]
Sorry to say, but looking at the responses there, I am not very confdent getting answers there.
See https://help.nextcloud.com/t/automatic-disconnection-log-out/82230
Secondly, I think there really is something wrong, or this should be documtented better (if I knew how, I would do that).

**Reply by szaimen** [reply]
Sorry, I fear I cannot help with this topic as I am not familiar with it. That is why I recommended to open a thread at https://help.nextcloud.com

**Reply by szaimen** [reply, marked as answer]
You could try to open an issue at https://github.com/nextcloud/server. Possibly you will get some help there.

**Reply by roelofz** [reply]
I followed up the thread where earlier issues were named to be fixed: https://github.com/nextcloud/server/issues/45132
Or should I open a new issue (don't know what is more appropriate).

**Reply by szaimen** [reply]
Probably better to create a new one
